1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,47 @@
-function setAlarm() {}
+let globSetinterval;
+let isPaused = false;
+let timer;
+let min;
+let sec;
+
+function setAlarm() {
+  let timeRemaining = document.getElementById("timeRemaining");
+  let alarmSet = document.getElementById("alarmSet").value *60;
+    if (alarmSet > 0) {
+      timer = setInterval(() =>{
+
+    if (!isPaused){
+      alarmSet -- ;
+      let min = Math.floor(alarmSet/60);
+      let sec = alarmSet % 60;
+      sec = sec < 10 ? `0${sec}`: sec;
+      min = min < 10 ? `0${min}`: min;
+      timeRemaining.textContent=`Time Remaining : ${min}: ${sec}`;
+    }
+    if (min == 0 && sec == 0) {
```

Comment:
Can you see which `min `and `sec `are being used here?  And do they have values or are they undefined?  This test always fails so the alarm never sounds.